### PR TITLE
[doc]: added recovery period term + show_complement() usage

### PR DESCRIPTION
**Pull request by Inglezos**
## Related issues
This is related to #531 and #478.

## What was changed
Added "recovery period" term and described `.show_complement()` usage.

### CODE REVIEW

File: docs/markdown/TERM.md [hunk 1]
```diff
@@ -50,5 +50,15 @@ If for some reason tau would be set to more than 1 day, then many records would
...
+<dt><strong>recovery period</strong></dt>
+<dd>The time period between case confirmation and recovery (as it is subjectively defined per country).
+What we currently do is to calculate the difference between confirmed cases and fatal cases and try to match it to some recovered cases value in the future.
+
+We apply this method for every country that has valid recovery data and average the partial recovery periods in order to obtain a single (average) recovery period.
+During the calculations, we ignore time intervals that lead to very short (<7 days) or very long (>90 days) partial recovery periods, if these exist with high frequency (>50%) in the records.
+We have to assume temporarily invariable compartments for this analysis to extract an approximation of the average recovery period.
+
+Alternatively, we had tried to use linelist data to get precise value of recovery period (average of recovery date minus confirmation date for cases), but the number of records was too small.</dd>
+
```
Comment (lisphilar) [inline docs/markdown/TERM.md:62]: Thank you for your pull request.
This be different from my first proposal, but the following could be informative.
- add usage of `JHUData.calculate_recovery_period()` to example/usage_dataset.ipynb
- add "Recovery period is the time period between..., but the number of records was too small." (all sentences you added to TERM.md) to example/usage_dataset.ipynb
- remove "What we currently do is to calculate..., but the number of records was too small." (all sentences except for the exact definition of recovery period) from TERM.md

File: example/usage_quick.ipynb [hunk 2]
```diff
@@ -142,6 +142,22 @@
...
+    "`Scenario.show_complement()` method is useful to show the kinds of complement. The details of complement are explained in [Usage: datasets](https://lisphilar.github.io/covid19-sir/usage_dataset.html) section."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "complement_snl_df = snl.show_complement()"
```
Comment (lisphilar) [inline example/usage_quick.ipynb:158]: The dataframe seems not shown. Please remove `complement_snl_df = `from this line.

File: example/usage_dataset.ipynb [hunk 3]
```diff
@@ -219,6 +219,37 @@
...
+   "metadata": {},
+   "source": [
+    "We can show the specific kind of complements that were applied to the records of each country with `JHUData.show_complement()` method."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The possible kinds of complement for each country are the following: \"Monotonic_confirmed\", \"Monotonic_fatal\", \"Monotonic_recovered\", \"Full_recovered\" and \"Partial_recovered\"."
```
Comment (lisphilar) [inline example/usage_dataset.ipynb:233]: Could you add desctriptive names and summaries of complement solutions, like this (could be changed)?
```
1. "Monotonic_*": monotonic increasing complment
Force the variable show monotonic increasing.

2. "Full_recovered": fully complement of recovered data
Estimate the number of recovered cases with the value of recovery period.

3. "Partial_recovered": partial complement of recovered data
When recovered values are not updated for some days, extrapolate the values.
```

**Review by lisphilar — CHANGES_REQUESTED** [review]
Thank you for your pull request. I added some comments.

File: example/usage_quick.ipynb (hunk 2 above)
Comment (Inglezos) [reply example/usage_quick.ipynb:158]: Added `complement_snl_df.tail()`.

**Review by lisphilar — APPROVED** [review]

**Pull request merged**